website/source/docs/configuration/plugin.html.md

```diff
@@ -17,7 +17,7 @@ description: |-
   </tr>
 </table>
 
-The `plugin` stanza is used to configure an individual plugin.
+The `plugin` stanza is used to configure an individual plugin. You may use this section to introduce new task or device drivers or simply modify their options.
```

Comment:
I don't think you can use the `plugin` stanza to introduce/install/enable plugins. I think they're either auto-detected in the plugin directory or not?

```suggestion
The `plugin` stanza is used to configure plugins.
```